### PR TITLE
rbd:show read:write proportion in the infomation of readwrite bench test

### PR DESCRIPTION
**Pull request by PCzhangPC**
Signed-off-by: PCzhangPC <pengcheng.zhang@easystack.cn>

### CODE REVIEW

**Comment by PCzhangPC** [comment]
@dillaman please help to review, thx

**Comment by PCzhangPC** [comment]
@dillaman please have a look at this small clean-up , thx

File: src/tools/rbd/action/Bench.cc [hunk 1]
```diff
@@ -212,6 +212,8 @@ int do_bench(librbd::Image& image, io_type_t io_type,
   std::cout << "bench "
        << " type " << (io_type == IO_TYPE_READ ? "read" :
                        io_type == IO_TYPE_WRITE ? "write" : "readwrite")
+       << (io_type == IO_TYPE_RW ? " read:write=" +
+           to_string(read_proportion) + ":" + to_string(100 - read_proportion) : "" )
```
Comment (dillaman) [inline src/tools/rbd/action/Bench.cc]: Nit: extra space before closing parenthesis

Comment (PCzhangPC) [reply src/tools/rbd/action/Bench.cc]: updated already

**Comment by PCzhangPC** [comment]
@ceph-jenkins retest this please

**Pull request merged**